app/src/main/java/org/mozilla/focus/utils/SupportUtils.java

```diff
@@ -36,4 +36,9 @@ public static String getSumoURLForTopic(final Context context, final String topi
 
         return "https://support.mozilla.org/1/mobile/" + appVersion + "/" + osTarget + "/" + langTag + "/" + escapedTopic;
     }
+
+    public static String getManifestoURL() {
+        final String langTag = Locales.getLanguageTag(Locale.getDefault());
+        return "https://www.mozilla.org/" + langTag + "/about/manifesto/";
```

Comment:
I wonder if there are locales this site does not support? If I use a made up locale then I see a 404 page but I can't find a valid locale that doesn't work.